meta/dalley-it/contao-photoswipe/en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+    title: Dalley IT - PhotoSwipe
+    description: >
+        Integration of javacsript library photoswipe. Easily adjust your photoswipe galleries in the contao backend.
```

Comment:
Typos: JavaScript, PhotoSwipe, Contao, back end

Reply:
Thank you. 
According to the error.log of travis the Typos are Dalley and PhotoSwipe. However your suggestions seem reasonable to me.

Can anyone please give me a hint how to extend the according whitelist.

Sorry I am new to this.